pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
Did I LGTM that doc? if so, bad me. I maintain the doc is wrong. We don't need that many levels of hierarchy.